src/test/java/HomePageTests.java

```diff
@@ -0,0 +1,71 @@
+import Pages.HomePage;
+import Pages.LoginPage;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+
+public class HomePageTests extends BaseTest{
+    String newPlaylistName = "Updated playlist";
+    @Test
+    public void renamePlaylist() throws InterruptedException {
+        LoginPage loginPage = new LoginPage(driver);
+        HomePage homePage = new HomePage(driver);
+        loginPage.login();
+//        Assert.assertTrue(homePage.getUserAvatar().isDisplayed());
+        Thread.sleep(2000);
+        homePage.doubleClickPlaylist().changePlaylistInputField();
+//        Assert.assertTrue(homePage.doesPlaylistExist().contains(newPlaylistName));
+
+    }
+
+
+
+    //public void doubleClickPlaylist() {
+        //WebElement choosePlayList = wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector(".playlist:nth-child(3)")));
+       // actions.doubleClick(choosePlayList).perform();
+
+   // }
+    //public void changePlaylistInputField(){
+        //WebElement playListInputField = wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("[name='name']")));
+        //playListInputField.sendKeys(Keys.chord(Keys.CONTROL, "a", Keys.BACK_SPACE));
+        //playListInputField.sendKeys("NewPlaylistName");
+        //playListInputField.sendKeys(Keys.ENTER);
+    //}
+
+    //public String doesPlaylistExist() {
+        //WebElement playlistElement = wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("div.success.show")));
+        //return playlistElement.getText();
+   // }
```

Comment:
I recommend not using code comments: for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.